azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/tools/auth/model/MavenAuthConfiguration.java

```diff
@@ -22,4 +28,39 @@
     private String serverId;
     private String httpProxyHost;
     private String httpProxyPort;
+
+    public void validate() throws InvalidConfigurationException {
+        String messagePrefix = null;
+        if (StringUtils.isBlank(tenant)) {
+            messagePrefix = "Cannot find 'tenant'";
+        } else if (StringUtils.isBlank(client)) {
+            messagePrefix = "Cannot find 'client'";
+        } else if (StringUtils.isAllBlank(key, certificate)) {
+            messagePrefix = "Cannot find either 'key' or 'certificate'";
+        } else if (StringUtils.isNotBlank(key) && StringUtils.isNotBlank(certificate)) {
+            messagePrefix = "It is wrong to specify both 'key' and 'certificate'";
```

Comment:
it seems this message is wrong.
and seems the upper 2 `if`s can be merged into one
```
if(StringUtils.isAllBlank(key, certificate) || StringUtils.isNoneBlank(key, certificate)){
    messagePrefix = "You have to specify either `key` or `certificate`";
}
```

Reply:
if merged the error message contains two instructions, which is not intuitive